pytest_splunk_addon/standard_lib/data_models/Alerts.json

```diff
@@ -1,75 +1,128 @@
+                },
+                {
+                    "name": "dest",
+                    "type": "required",
+                    "comment": "The destination of the alert message, such as an email address or SNMP trap. You can alias this from more specific fields, such as dest_host, dest_ip, or dest_name."
+                },
+                {
+                    "name": "dest_type",
+                    "type": "optional",
+                    "comment": "The type of the destination object, such as instance, storage, firewall."
+                },
+                {
+                    "name": "id",
+                    "type": "required",
+                    "comment": "The unique identifier of a message."
+                },
+                {
+                    "name": "mitre_technique_id",
+                    "type": "optional",
+                    "comment": "The MITRE ATT&CK technique ID of the alert event."
+                },
+                {
+                    "name": "signature",
+                    "type": "optional",
+                    "comment": "A human-readable signature name."
+                },
+                {
+                    "name": "signature_id",
+                    "type": "optional",
+                    "comment": "The unique identifier or event code of the event signature."
+                },
+                {
+                    "name": "severity",
+                    "type": "required",
+                    "expected_values": [
+                        "critical",
+                        "high",
+                        "medium",
+                        "low",
+                        "informational"
+                    ],
+                    "comment": "The severity of a message."
+                },
+                {
+                    "name": "severity_id",
+                    "type": "optional",
+                    "comment": "A numeric severity indicator for a message."
+                },
+                {
+                    "name": "src",
+                    "type": "optional",
+                    "comment": "The source of the message. You can alias this from more specific fields, such as src_host, src_ip, or src_name."
+                },
+                {
+                    "name": "src_type",
+                    "type": "optional",
+                    "comment": "The type of the source object, such as instance, storage, firewall."
+                },
+                {
+                    "name": "subject",
+                    "type": "optional",
+                    "comment": "The message subject."
+                },
+                {
+                    "name": "type",
+                    "type": "required",
+                    "expected_values": [
+                        "alarm",
+                        "alert",
+                        "event",
+                        "task",
+                        "warning"
```

Comment:
`unknown` is missing here?

Reply:
This was not added, reason can be found here: https://github.com/splunk/pytest-splunk-addon/pull/595#discussion_r840602600

Reply:
this is in line with our 'don't map null values' policy